graph/api/src/main/java/org/jboss/windup/graph/GraphTypeManager.java

```diff
@@ -270,21 +270,21 @@ public static boolean hasType(Class<? extends WindupVertexFrame> type, Vertex v)
         return false;
     }
 
-    public static AbstractElement asTitanVertex(Element e)
+    public static AbstractElement asTitanElement(Element e)
     {
         if (e instanceof StandardVertex)
         {
             return (StandardVertex) e;
         }
         else if (e instanceof EventVertex)
         {
-            return (StandardVertex) ((EventVertex) e).getBaseVertex();
+            return (AbstractElement) ((EventVertex) e).getBaseVertex();
         } else if (e instanceof StandardEdge)
```

Comment:
Note that the original code casted to Titan's StandardVertex too. Is this solving some CCEx?